geoportal/c2cgeoportal_geoportal/scaffolds/create/geoportal/vars.yaml_tmpl

```diff
@@ -45,6 +45,12 @@ vars:
           coordinatesProjections: *alternate_projections
         gmfPermalinkOptions:
           projectionCodes: *alternate_projections
+        gmfVectorTilesOptions:
+          projection: EPSG:{srid}
+          tileGrid:
+            extent: *extent
+            origin: [2420000, 1350000]
```

Comment:
Why is it needed?

Reply:
To make a correct TileGrid

Reply:
But it can be extracted from the extent, no?

Reply:
In Openlayers, we can define extent and origin independently. Origins looks to be the point 0/0/z of the grid, and extent a mask to not display the tiles outside it.
I've kept the original Openlayers TileGrid possibilities. Eventually, I can extract the origins from the extent if the origin is not defined. But I was keeping this poc as low budget as possible.